### PR TITLE
InfluxDB 2.x support

### LINKED ISSUE #430: Upgraded to influx v2 and some data is now missing.
I updated my influxdb to v2 and noticed that now some graphs are empty (UAP Bandwith, Received signal strength, wireless signal  and many others).  No errors reported on the service:

    [INFO] UniFi Metrics Recorded. Site: 1, Client: 58, UAP: 2, UDM/USG/: 0, USW: 3, IDS/Event/Alarm/Anomaly: 0/0/0/0, DPI Site/Client: 0/0, Point: 110, Fields: 3360, Err: 0, Dur: 13ms

I do have some data such as switch temperature, fan level, memory, data transfer and others. 

Any ideas?

### PR DESCRIPTION
**Pull request by platinummonkey**
- Keeps older influxdb <1.8 under `influx_v1_unifi`
- Add new option for influxdb >=1.8, 2.x `influx_v2_unifi`

fixes #430 #407 #255

### CODE REVIEW

File: pkg/influx_v1_unifi/events.go [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+	"github.com/unpoller/unifi"
+)
+
+// These constants are used as names for printed/logged counters.
+const (
+	eventT = item("Event")
+	idsT   = item("IDS")
+)
+
+// batchIDS generates intrusion detection datapoints for InfluxDB.
```
Comment (davidnewhall) [inline pkg/influx_v1_unifi/events.go:15]: is this a n ew file? The history going poof at this point?

Comment (davidnewhall) [reply pkg/influx_v1_unifi/events.go:15]: or did you duplicate the files, and it's seeing the copy in the wrong folder?

Comment (platinummonkey) [reply pkg/influx_v1_unifi/events.go:15]: It’s a git move but GitHub seems to be flailing. I can separate these PRs though

**Comment by platinummonkey** [comment]
actually going to merge these, closing in favor of another PR incoming

**Pull request closed without merging**